rfd/0162-machine-id-token-join-method-bot-instance.md

```diff
@@ -0,0 +1,653 @@
+---
+authors: Noah Stride (noah.stride@goteleport.com)
+state: draft
+---
+
+# RFD 00162 - Improving the on-prem fleet Bot management experience
+
+## Required approvers
+
+- Engineering: @zmb3
+- Product: @klizhentas || @xinding33
+- Security: @reedloden || @jentfoo
+
+## What
+
+Terminology:
+
+- Bot: An identity within Teleport intended for use by machines as opposed to
+  humans. Many individual machines may act as this shared identity.
+- `tbot`: The Teleport binary that acts as aBot and generates credentials for
+  consumption by client applications.
+- Bot instance: A single instance of `tbot` running on a host.
+
+This RFD proposes improvements to the management of fleets of Machine ID Bots.
+These improvements are mostly targetted at on-prem deployments, where the
+delegated join methods are not available.
+
+The improvements will focus on three points:
+
+- To allow multiple Bot instances to be associated with a single Bot when using
+  the `token` join method.
+- To allow multiple Bot instances to be joined using a single join token when
+  using the `token` join method.
+- Providing a way to track and monitor Bot instances.
+
+## Why
+
+Whilst deploying a large fleet of Bots is fairly trivial when using the
```

Comment:
I might mention TPM joining here as well. If TPMs are available, that flow is probably preferred.